modules/nvidia-gpu-prerequisites.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// * architecture/nvidia-gpu-architecture-overview.adoc
+
+
+:_content-type: CONCEPT
+[id="nvidia-gpu-prerequisites_{context}"]
+= NVIDIA GPU prerequisites
+
+* A working OpenShift cluster up and running with a GPU worker node.
+
+* Access to the OpenShift cluster as a `cluster-admin` to perform the necessary steps.
+
+* OpenShift CLI (`oc`) installed.
+
+* NFD operator must be installed along with an instance of ``nodefeaturediscovery``. 
```

Comment:
It sounds a bit awkward to me. An instance of `NodeFeatureDiscovery` is created after the operator has been installed. [Here](https://docs.openshift.com/container-platform/4.13/hardware_enablement/psap-node-feature-discovery-operator.html#using-the-node-feature-discovery-operator_node-feature-discovery-operator) is how the OpenShift doc puts it.

Reply:
Reworded: "Install the Node Feature Discovery (NFD) operator and create a ``nodefeaturediscovery`` instance."